README.md

```diff
@@ -65,34 +64,49 @@ npm run server
 This will create the `auspice/index.html` and `auspice/dist/*` files which are gitignored.
 Note that the favicon.png isn't needed for auspice, as the nextstrain.org server handles this.
 
+You can also run Auspice's own development server (not the nextstrain.org server) to ease development of client customizations.
+This means that you will see a different splash page & the documentation will not work.
+It is not currently possible to run auspice in development mode & see the nextstrain.org splash page & docs.
+Read the section below on the nextstrain.org server for more context.
+
+```bash
+cd auspice
+npx auspice develop --verbose --extend ./client/config.json --handlers ./server/index.js
+```
+
+> If you don't have [`npx`](https://www.npmjs.com/package/npx) installed, you can either install it first with `npm install --global npx` or just use `../node_modules/.bin/auspice` in its place.
```

Comment:
I don't think this is necessary. `npx` comes with `npm` starting at version 5.2 (~2 years ago). Our install instructions are for `nodejs=10` which will bring in (at least) npm 5.8. If a user doesn't have `npx` then I think they should be advised to update node & npm.

Reply:
Oh, that's great!

Reply:
Pushed an adjustment to this as a50fdf0.